dataproc_jupyter_plugin/services/dataproc.py

```diff
@@ -18,9 +18,15 @@
     DATAPROC_SERVICE_NAME,
 )
 
+from google.cloud import dataproc_v1 as dataproc
+import proto
+import json
```

Comment:
This import order is wrong.

Standard library imports (e.g. `json`) have to go first, then external imports (`google.cloud`, `google.oauth2.credentials`, `google.protobuf.empty_pb2`), and finally local packages (`dataproc_jupyter_plugin` and it's sub packages).

Within each section, the imports should be in alphabetical order unless the imports have side effects that must be performed in a specific order.